srcpkgs/g15daemon/files/g15daemon/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
+exec g15daemon
```

Comment:
Readme says that it needs to be root for logging, but logging by unprivileged user do work. Any problems with running as dedicated user crated as in [Manual.md](https://github.com/void-linux/void-packages/blob/master/Manual.md#creating-system-accountsgroups-at-runtime) and used as in [netdata](https://github.com/void-linux/void-packages/blob/189d87cab913dfca5bb5b33ea37c0965d4329264/srcpkgs/netdata/files/netdata/run)?